public/app/features/alerting/unified/home/Insights.tsx

```diff
@@ -0,0 +1,160 @@
+        getInvalidConfigScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Invalid configuration'),
+      ],
+    }),
+  });
+}
+
+function getMimirManagedRulesScenes() {
+  return new EmbeddedScene({
+    body: new SceneFlexLayout({
+      wrap: 'wrap',
+      children: [
+        getMostFiredCloudInstances(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Top 10 firing instance this week'),
+        getFiringCloudAlertsScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Firing'),
+        getPendingCloudAlertsScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Pending'),
+
+        getInstancesByStateScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Count of alert instances by state'),
+        getInstancesPercentageByStateScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, '% of Alert Instances by State'),
+
+        getEvalSuccessVsFailuresScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Evaluation Success vs Failures'),
+        getMissedIterationsScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Iterations Missed'),
+      ],
+    }),
+  });
+}
+
+function getMimirManagedRulesPerGroupScenes() {
+  const ruleGroupHandler = new QueryVariable({
+    label: 'Rule Group',
+    name: 'rule_group',
+    datasource: cloudUsageDs,
+    query: 'label_values(grafanacloud_instance_rule_group_rules,rule_group)',
+  });
+
+  return new EmbeddedScene({
+    body: new SceneFlexLayout({
+      wrap: 'wrap',
+      children: [
+        getRuleGroupEvaluationsScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Rule Group Evaluation'),
+        getRuleGroupIntervalScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Rule Group Interval'),
+        getRuleGroupEvaluationDurationScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Rule Group Evaluation Duration'),
+        getRulesPerGroupScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Rules per Group'),
+      ],
+    }),
+    $variables: new SceneVariableSet({
+      variables: [ruleGroupHandler],
+    }),
+    controls: [new VariableValueSelectors({})],
+  });
+}
+
+export default function Insights() {
+  const styles = useStyles2(getStyles);
+  const grafanaScenes = getGrafanaScenes();
+  const cloudScenes = getCloudScenes();
+  const mimirRulesScenes = getMimirManagedRulesScenes();
+  const mimirRulesPerGroupScenes = getMimirManagedRulesPerGroupScenes();
+
+  return (
+    <div className={styles.container}>
+      <CollapsableSection label="Grafana" isOpen={true}>
+        <grafanaScenes.Component model={grafanaScenes} />
+      </CollapsableSection>
+
+      <CollapsableSection label="Mimir Alertmanager" isOpen={true}>
+        <cloudScenes.Component model={cloudScenes} />
+      </CollapsableSection>
+
+      <CollapsableSection label="Mimir-managed Rules" isOpen={true}>
+        <mimirRulesScenes.Component model={mimirRulesScenes} />
+      </CollapsableSection>
+
+      <CollapsableSection label="Mimir-managed Rules - Per Rule Group" isOpen={true}>
+        <mimirRulesPerGroupScenes.Component model={mimirRulesPerGroupScenes} />
+      </CollapsableSection>
+    </div>
```

Comment:
I would try to break the scene dashboard up into separate tabs so you do not have to scroll so much, but just a thought